AudioBar/AudioBarViewController.swift

```diff
@@ -82,12 +90,16 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
                     beginObservingPlayerItem(playerItem)
                     player.replaceCurrentItem(with: playerItem)
                 }
+            case .start(withDuration: let duration):
+                player.play()
+                nowPlayingInfoCenter.nowPlayingInfo![MPMediaItemPropertyPlaybackDuration] = duration
```

Comment:
Why isn't this done declaratively in `didUpdate view`? Also, `.player` messages are for the player. It's in the name! Player only.